app/api/helpers/permission_manager.py

```diff
@@ -161,6 +162,31 @@ def is_speaker_for_session(view, view_args, view_kwargs, *args, **kwargs):
     return ForbiddenError({'source': ''}, 'Access denied.').respond()
 
 
+@jwt_required
+def is_speaker_via_cfs(view, view_args, view_kwargs, *args, **kwargs):
```

Comment:
is_speaker_itself_or_admin

Reply:
However, there is already a check for admin, so just make this for speaker

Don't repeat functionalities

Reply:
@iamareebjamal I will rename it alright,  but we need to permit admin, organisers and the user itself . Correct?

Reply:
You can apply multiple annotations to the dame function. Each should handle only its own task, not multiple